modules/api_reference/devportal_api_reference.install

```diff
@@ -151,3 +165,54 @@ function devportal_api_reference_update_8001(): ?string {
   }
   return t('Swagger UI enabled for API Reference nodes.');
 }
+
+/**
+ * Change machine name of the display of the Devportal API Reference view.
+ */
+function devportal_api_reference_update_8002(): ?string {
+  $database = \Drupal::database();
+  // Start a DB transaction in order to have some hopefully meaningful rollback
+  // option if anything goes wrong.
+  $transaction = $database->startTransaction();
+  try {
+    $view = View::load('devportal_api_reference');
+    // Remove page_1 display.
+    $displays = $view->get('display');
+    unset($displays['page_1']);
+    // Add admin_api_reference_list.
+    $displays['admin_api_reference_list'] = [
+      'display_plugin' => 'page',
+      'id' => 'admin_api_reference_list',
+      'display_title' => 'Page',
+      'position' => 1,
+      'display_options' => [
+        'display_extenders' => [],
+        'path' => 'admin/devportal/api-reference',
+      ],
+      'cache_metadata' => [
+        'max-age' => 0,
+        'contexts' => [
+          'languages:language_content',
+          'languages:language_interface',
+          'url',
+          'url.query_args',
+          'user.node_grants:view',
+          'user.permissions'
+        ],
+        'tags' => [
+          'config:field.storage.node.field_version'
+        ],
+      ],
+    ];
+    $view->set('display', $displays);
+    $view->save();
+  }
+  catch (Exception $e) {
+    // Catch any exceptions that are thrown.
+    $transaction->rollBack();
+    Drupal::messenger()
+      ->addError(t('@message', ['@message' => $e->getMessage()]));
```

Comment:
Well, there is nothing to translate here.... you could provide some extra information where is this error message is coming from.